Reference channels for newcomer contributors, #jenkins-release and Slack in the /chat page (#4226)
jenkins-infra/jenkins.io: content/chat.adoc

General discussions about Jenkins.

=== https://gitter.im/jenkinsci/newcomer-contributors[jenkinsci/newcomer-contributors]

Q&A channel for newcomer contributors to Jenkins.

> Use this channel if you are not sure where to ask, and other contriburos will help you.

Reviewer: ```suggestion
Use this channel if you are not sure where to ask, and other contributors will help you.
```

* email jieryn@gmail.com with those details
* monitor your memoserv notices for updates from Freenode staff requesting confirmation of cloak request
** */msg memoserv help*

== Slack

Some teams and working groups use Slack for communications.

=== Continuous Delivery Foundation 

link:https://cd.foundation/[Continuous Delivery Foundation (CDF)] Slack is used for coordinating the Jenkins community activities with the CDF, other projects and special interest groups.
Any contributor can join this channel, but the Jenkins community does not currently host its official channels there.
link:https://join.slack.com/t/cdeliveryfdn/shared_invite/zt-nwc0jjd0-G65oEpv5ynFfPD5oOX5Ogg[Join CDF Slack]

=== Kubernetes

> There is an **unofficial** `#jenkins-ci` channel in the Kubernetes Slack.

Reviewer: ```suggestion
The Kubernetes Slack includes an **unofficial** `#jenkins-ci` channel.
```
